cfgmgr/intfmgr.cpp

```diff
@@ -517,6 +521,12 @@ bool IntfMgr::doIntfGeneralTask(const vector<string>& keys,
                 FieldValueTuple fvTuple("nat_zone", nat_zone);
                 data.push_back(fvTuple);
             }
+
+            if (!ipv6_link_local_mode.empty())
```

Comment:
HLD mentions about setting the sysctl variable - `sysctl -w net.ipv6.conf.default.disable_ipv6=1`
Where is this handled. Also, could you please revise the HLD based on the implementation. The flows are different b/w HLD and implementation.

Reply:
Updated the HLD. Setting sysctl variable is not needed as IPv6 is enabled by default as per new design.